test/helpers/fakeFactory.js

```diff
@@ -0,0 +1,24 @@
+var _ = require('underscore');
```

Comment:
👏  We should start using like this more frequently...

Reply:
He´s phony... she's `fake`... that's the type of people I hate. https://www.youtube.com/watch?v=_T2yTDxJOdQ

Reply:
yep, imo we should have a .spec.js and a .mock.js for every file :)

btw: I like to use [sinon](http://sinonjs.org/) for the spies/mocks it´s a nice tool!